Logger.Tests/LogFactoryTests.cs

```diff
@@ -1,11 +1,52 @@
 ﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
 using System.IO;
 
-namespace Logger.Tests
-{
+namespace Logger.Tests;
+
     [TestClass]
     public class LogFactoryTests
     {
+        [TestMethod]
+        public void CreateLogger_GivenValidFilePath_ReturnsFileLogger()
+        {
+            // Arrange
+            var logFactory = new LogFactory();
+            string filePath = Path.GetFullPath(Path.GetRandomFileName());
+            logFactory.ConfigureFileLogger(filePath);
+
+            // Act
+            BaseLogger logger = logFactory.CreateLogger(nameof(LogFactoryTests));
+
+            // Assert
+            Assert.IsTrue(logger is FileLogger);
+            Assert.AreEqual(nameof(LogFactoryTests), logger.ClassName);
+        }
+
+        [TestMethod]
+        public void CreateLogger_GivenNullFilePath_ReturnFalse()
```

Comment:
```suggestion
        public void CreateLogger_GivenNullFilePath_ReturnsNull()
```
This may be a better name.